### PR TITLE
Update FAQ, mention gateway Switzerland

### PR DESCRIPTION
**Pull request by dsarkar**
See comment from @MikeMcC399:  https://github.com/corona-warn-app/cwa-website/issues/1060#issue-853454573, point 2.

### CODE REVIEW

**Comment by dsarkar** [comment]
image

File: src/data/faq_de.json [hunk 1]
```diff
@@ -444,7 +444,7 @@
...
+                            "Gemeinsam mit der EU-Kommission wurde eine Infrastruktur für einen sicheren Informationsaustausch entwickelt, die als sog. \"Interoperability Gateway\" bezeichnet wird. Dieses Gateway ermöglicht, dass die Corona-Warn-App über Landesgrenzen hinaus funktioniert und auch im Ausland potentielle Warnmeldungen empfangen werden können. Detaillierte Informationen zum Gateway bietet die EU-Kommission unter <a href='https://ec.europa.eu/commission/presscorner/detail/de/qanda_20_1905#gateway'>Coronavirus: EU-Datenabgleichsdienst für Kontaktnachverfolgungs- und Warn-Apps – Fragen und Antworten</a>. Damit auch Warnungen zwischen Nutzern der Schweizer Corona-App und der Corona- Warn-App möglich sind, betreibt das RKI zudem gemeinsam mit der Schweiz (Bundesamt für Gesundheit der Schweizerischen Eidgenossenschaft) einen weiteren Austausch-Server.",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:447]: "Corona- Warn-App" => "Corona-Warn-App"

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
There is an extra space which should be removed in the middle of the app name "Corona- Warn-App" in the German text.

**Comment by MikeMcC399** [comment]
Also for consistency, you might want to change the app name in the title from "Corona Warn App" to "Corona-Warn-App".

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks a lot!

**Review by MikeMcC399 — APPROVED** [review]
LGTM!

**Pull request merged**